### PR TITLE
Update workflow-commands-for-github-actions.md

### PR DESCRIPTION
**Pull request by MayCXC**
### Why:

The behavior I currently see in workflows is that outputs are not present when not defined in a job, like they were in this example.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

workflow docs example

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by MayCXC** [comment]
reopening after testing more, looks like the job outputs are empty when set up like this example, and it contradicts: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

**Comment by Coinsintegrity** [comment]
> ### Por qué:
> El comportamiento que veo actualmente en los flujos de trabajo es que las salidas no están presentes cuando no están definidas en un trabajo, como en este ejemplo.
> 
> ### Qué se está cambiando (si está disponible, incluya fragmentos de código, capturas de pantalla o gifs):
> ejemplo de documentos de flujo de trabajo
> 
> ### Marque lo siguiente:
> * [x] He revisado mis cambios en la puesta en escena (busque el "Comentario generado automáticamente" y haga clic en los enlaces en la columna "Vista previa" para ver sus últimos cambios).
> * [x] Para los cambios de contenido, he completado la [lista de verificación de autoevaluación](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review) .
Bueno

**Comment by Coinsintegrity** [comment]
Buenos argumentos

**Review by rsese — DISMISSED** [review]
change makes sense to me 👍 

for this bit further down in the next job to work:

```
        SECRET_HANDLE="${{ needs.secret-generator.outputs.handle }}"{% endraw %}
```

the `handle` output needs to be defined in the `secret-generator` job

File: content/actions/using-workflows/workflow-commands-for-github-actions.md [hunk 1]
```diff
@@ -496,12 +496,15 @@ on: push
 jobs:
   secret-generator:
     runs-on: ubuntu-latest
+    outputs:
+      handle: ${{ steps.generate-secret.outputs.handle }}
```
Comment (sabrowning1) [inline content/actions/using-workflows/workflow-commands-for-github-actions.md:500]:
```suggestion
      handle: {% raw %}${{ steps.generate-secret.outputs.handle }}{% endraw %}
```

We should add `{% raw %}` and `{% endraw %}` tags here so the [bracketed content renders properly](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#code-blocks) 🙂

**Review by sabrowning1 — CHANGES_REQUESTED** [review]
Thanks again for catching this and opening a PR @MayCXC! ✨ I've requested one small formatting change below, then this should be good to go.

**Comment by peterbe** [comment]
Hi @MayCXC Thanks for your contribution!
Can you heed @sabrowning1 's suggestion and then I think we can finish the review of this. Thanks in advance!

**Review by sabrowning1 — APPROVED** [review]
👋🏼 @MayCXC, I've committed my suggestion and will merge this shortly. Thanks again for your help maintaining our docs! ✨

**Pull request merged**